doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt

```diff
@@ -0,0 +1,12 @@
+.. code-block:: bash
+
+ #Get the backup name
+ $ pbm list
+ Backup history:
+   2020-04-20T10:55:42Z
+   2020-04-20T13:07:34Z
+   2020-04-20T13:13:20Z
```

Comment:
```suggestion
   2020-04-20T13:13:20Z
   2020-04-20T13:07:34Z
   2020-04-20T10:55:42Z
```
The order of backups has been changed by https://jira.percona.com/browse/PBM-438. The top of the list will be the most recent now.

Reply:
Actually no. The most recent remains in the bottom. PBM-438 didn't change the sorting order but fixed output slice fo `pbm list --size`